docs/rancher/node/node-driver.md

```diff
@@ -118,7 +118,7 @@ During the cluster installation, the Harvester node driver will automatically he
 
 :::note
 
-Label synchronization will only take effect during guest node initialization. To avoid node drifts to another region or zone, it is recommended to add the [node affinity rules](./rke2-cluster.md#add-node-affinity) during the cluster provisioning, so that the VMs can be scheduled to the same zone even after rebuilding.
+When a migration or change occurs for the VM corresponding to the guest node, the topology labels on the Harvester node will be automatically resynchronized. To avoid node drifts to another region or zone, it is recommended to add the [node affinity rules](./rke2-cluster.md#add-node-affinity) during the cluster provisioning, so that the VMs can be scheduled to the same zone even after rebuilding.
```

Comment:
```suggestion
When a migration or change occurs for a VM corresponding to the guest node, the topology labels on the Harvester node automatically resynchronize. To avoid node drifts to another region or zone, we recommend adding [node affinity rules](./rke2-cluster.md#add-node-affinity) during the cluster provisioning, allowing you to schedule VMs to the same zone even after rebuilding.
```